ui/app/components/secret-engine/configure-aws.hbs

```diff
@@ -109,4 +120,23 @@
       />
     {{/if}}
   </div>
-</form>
\ No newline at end of file
+</form>
+
+{{#if this.saveIssuerWarning}}
+  <Hds::Modal id="rotate-credentials-modal" @onClose={{this.cancelSave}} data-test-issuer-warning as |M|>
+    <M.Header @icon="info">
+      Save global issuer?
+    </M.Header>
+    <M.Body>
+      <p class="has-bottom-margin-s">
+        {{this.saveIssuerWarning}}
+      </p>
+    </M.Body>
+    <M.Footer>
+      <Hds::ButtonSet>
+        <Hds::Button @text="Continue" {{on "click" (action (mut this.saveIssuerWarning) false)}} data-test-save-issuer />
+        <Hds::Button @text="Cancel" @color="secondary" {{on "click" this.cancelSave}} data-test-cancel />
```

Comment:
NIT: change the selector to something specific like the save button selector, `data-test-cancel-issuer`. I think I have another data-test-cancel selector on the template.